modules/nw-kuryr-migration.adoc

```diff
@@ -0,0 +1,349 @@
+
+. To start the migration, configure the OVN-Kubernetes network plug-in by using one of the following commands:
+
+** To specify the network provider without changing the cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
+----
+
+** To specify a different cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{
+    "spec": {
+      "clusterNetwork": [
+        {
+          "cidr": "<cidr>",
+          "hostPrefix": "<prefix>"
+        }
+      ]
+      "networkType": "OVNKubernetes"
+    }
+  }'
+----
++
+where:
+
+<cidr>:: Specifies a CIDR block.
+<prefix>:: Specifies a slice of the CIDR block that is apportioned to each node in your cluster.
++
+[IMPORTANT]
+====
+You cannot change the service network address block during the migration.
+
+You cannot use any CIDR block that overlaps with the `100.64.0.0/16` CIDR block because the OVN-Kubernetes network provider uses this block internally.
+====
+
+. Verify that the Multus daemon set rollout is complete by entering the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-multus rollout status daemonset/multus
+----
++
+The name of the Multus pods is in the form of `multus-<xxxxx>`, where `<xxxxx>` is a random sequence of letters. It might take several moments for the pods to restart.
++
+.Example output
+[source,text]
+----
+Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
+...
+Waiting for daemon set "multus" rollout to finish: 5 of 6 updated pods are available...
+daemon set "multus" successfully rolled out
+----
+
+. To complete the migration, reboot each node in your cluster. For example, you can use a bash script similar to the following example. The script assumes that you can connect to each host by using `ssh` and that you have configured `sudo` to not prompt for a password.
++
+[source,bash]
+----
+#!/bin/bash
+
+for ip in $(oc get nodes  -o jsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
+do
+   echo "reboot node $ip"
+   ssh -o StrictHostKeyChecking=no core@$ip sudo shutdown -r -t 3
+done
+----
++
+[NOTE]
+====
+If ssh access is not available, you could alternatively use `openstack` command
+to do so:
+[source,terminal]
+----
+$ for name in $(openstack server list --name ${CLUSTERID}\* -f value -c Name); do openstack server reboot $name; done
+----
+It might also be able to reboot each node through the management portal for
+your infrastructure provider. Otherwise contact appropriate authority for
+either gaining access to the virtual machines through ssh or management
+portal/OpenStack client.
+====
+
+. Confirm that the migration succeeded, and then remove the migration resources:
```

Comment:
Not sure if I understand you.

Reply:
@gryf The idea is in the Procedure Verification section in https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

However, since it seems like the cleanup steps are mandatory, I wouldn't change this at this time.